doc/rados/operations/pools.rst

```diff
@@ -96,60 +102,49 @@ Where:
 
   :Type: Integer
-  :Required: No. Set to ``1`` if autoscaling is enabled, otherwise picks up Ceph
-         configuration value :confval:`osd_pool_default_pg_num`
-  :Default: Value of :confval:`osd_pool_default_pg_num`
+  :Required: Yes.
+  :Default: 8
 
 .. describe:: {pgp-num}
 
-   The total number of placement groups for placement purposes. This
-   **should be equal to the total number of placement groups**, except
-   for placement group splitting scenarios.
+   The total number of PGs for placement purposes. This
+   **should be equal to the total number of PGs**, except
+   in PG-splitting scenarios.
 
   :Type: Integer
-  :Required: No. Picks up Ceph configuration value :confval:`osd_pool_default_pgp_num`
-         if not specified. If that is not set, defaults to value of ``pg-num``.
-  :Default: Value of ``pg-num`` 
+  :Required: Yes. If no value has been specified in the command, then the default value is used (unless a different value has been set in Ceph configuration).
+  :Default: 8
 
 .. describe:: {replicated|erasure}
 
-   The pool type which may either be **replicated** to
-   recover from lost OSDs by keeping multiple copies of the
-   objects or **erasure** to get a kind of
-   `generalized RAID5 <../erasure-code>`_ capability.
-   The **replicated** pools require more
-   raw storage but implement all Ceph operations. The
-   **erasure** pools require less raw storage but only
-   implement a subset of the available operations.
+   The pool type. This can be either **replicated** (to recover from lost OSDs
+   by keeping multiple copies of the objects) or **erasure** (to achieve a kind
+   of `generalized RAID5 <../erasure-code>`_ capability).  The **replicated**
+   pools require more raw storage but can implement all Ceph operations. The
+   **erasure** pools require less raw storage but can implement only some Ceph
+   operations.
 
   :Type: String
   :Required: No.
   :Default: replicated
 
 .. describe:: [crush-rule-name]
 
-   The name of a CRUSH rule to use for this pool.  The specified
-   rule must exist.
+   The name of a CRUSH rule to use for this pool. The specified rule must
+   exist; otherwise the command will fail.
 
    :Type: String
    :Required: No.
-   :Default: For **replicated** pools it is the rule specified by the
-          :confval:`osd_pool_default_crush_rule` config variable.  This rule must exist.
-          For **erasure** pools it is ``erasure-code`` if the ``default``
-          `erasure code profile`_ is used or ``{pool-name}`` otherwise.  This
-          rule will be created implicitly if it doesn't exist already.
-
+   :Default: For **replicated** pools, it is the rule specified by the :confval:`osd_pool_default_crush_rule` configuration variable. This rule must exist.  For **erasure** pools, it is the ``erasure-code`` rule if the ``default`` `erasure code profile`_ is used or the ``{pool-name}`` rule  if not. This rule will be created implicitly if it doesn't already exist.
```

Comment:
The ghosts of Perkin Elmer Bantams weep at this long line

Reply:
I'll see if I can't wrap it.

Reply:
Update: Nope. The :conval: directive is interpreted as a new field. This is an ambiguity in the RST standard, so it'll have to be one giant line.

\<Strother Martin\>I don't like it any more than you. <Strother Martin>